docs/README.md

```diff
@@ -0,0 +1,21 @@
+# The rust-verificatio-tools blog
```

Comment:
Cool, thanks for writing this!

Can you move it to docs/_internal/howto-blog.md
so that it doesn't accidentally become part of the website (the _ part)
it is clear that it is documentation for us (the internal part)
and it's a bit easier to guess what it is about from the filename.

Reply:
Done